apps/site/lib/site/realtime_schedule.ex

```diff
@@ -227,6 +236,10 @@ defmodule Site.RealtimeSchedule do
   end
 
   @spec build_output(map, [route_with_patterns_t], map, map, map, DateTime.t()) :: [map]
+  defp build_output(%{message: _message}, _, _, _, _, _) do
```

Comment:
@ryan-mahoney it's not clear what this clause is for

Reply:
removed